src/components/Search/SearchDropdown.tsx

```diff
@@ -16,6 +16,11 @@ function SearchDropdown(props: SearchDropdownProps) {
   const { repository, view } = props;
   const { handleChangeDropdown } = useHandleChangeDropdown();
   const size = view == 'compare-results' ? 'small' : undefined;
+  const repoSelect = {
+    baseRepository: '',
+    newRepository: '',
+    searchType: 'base' as 'base' | 'new',
```

Comment:
Could we change `'base' | 'new'` to be a type or move them into constants?

Reply:
Yes, that makes sense. Thank you!